src/components/checkboxSet/__snapshots__/CheckboxSet.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CheckboxSet renders 1`] = `<CheckboxSetstyles__List />`;
```

Comment:
I'd just delete this file and the associated test. It's not really testing anything here.

Reply:
👍 Thank you, done this.